docs/cloud/pouta/create-jumphost.md

```diff
@@ -0,0 +1,41 @@
+# Jumphost in cPouta
+## Introduction
+The purpose of a jumphost in an infrastructure is to allow access from the external to several machines without giving them a public IP (Floating IP).  
+You can configure one machine with a public IP and access the other machines from the jumphost. This unique machine with a public IP is a gateway to the internal network.  
+
+Here is a schema:  
+![jumphost-schema](../img/jumphost_schema.png)
+
+
+## How to create a jumphost?
+You can find a template in our GitHub repo: [jumphost CSC](https://github.com/CSCfi/openshift-jumphost-example). Read carefully the instructions provided in the repo.  
+Keep in mind that we provide a basic template. You can clone and edit it for fulfilling your needs.
+
+## Explanations  
+Two extra folders need to be created: `files` and `keys`.  
+
+In the `files` folder, you need to copy your private and public used to connect the instances. It can be the public and private keys you use every time in cPouta or it can be a new key pair. Don't forget to upload your key to your cPouta project.  
```

Comment:
private?

Reply:
In principle you do not need a private key in the jumphost because it gets forwarded from your local disk.